shared-bindings/rgbmatrix/RGBMatrix.c

```diff
@@ -133,11 +133,11 @@ STATIC void preflight_pins_or_throw(uint8_t clock_pin, uint8_t *rgb_pins, uint8_
     }
 }
 
-//| :class:`~_protomatter.Protomatter` displays an in-memory framebuffer to an LED matrix.
+//| :class:`~rgbmatrix.RGBMatrix` displays an in-memory framebuffer to an LED matrix.
 //|
-//| .. class:: Protomatter(width, bit_depth, rgb_pins, addr_pins, clock_pin, latch_pin, output_enable_pin, *, doublebuffer=True, framebuffer=None)
+//| .. class:: RGBMatrix(width, bit_depth, rgb_pins, addr_pins, clock_pin, latch_pin, output_enable_pin, *, doublebuffer=True, framebuffer=None)
```

Comment:
I thought we added height here. Did it get lost in the PR shuffle?

Reply:
Docs got out of date.  Fixed now?